library/persistence/book-price.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class BookPrice:
+    price_per_book: float = 3.55
+    min_books_for_discount: int = 3
+    discount_per_book: float = 0.5
+    discount_per_reading_credit: float = 0.5
```

Comment:
Maybe one should make these class variables such that `BookPrice` does not to be instantiated to access theser?